lib/src/over_react_redux/hooks/use_selector.dart

```diff
@@ -129,11 +129,11 @@ import 'package:react/react_client/react_interop.dart' show ReactContext;
 /// > Related: [useDispatch]
 TValue useSelector<TReduxState, TValue>(
   TValue Function(TReduxState state) selector, [
-  bool Function(TValue tNextValue, TValue tPrevValue) equalityFn,
+  bool Function(TValue? tNextValue, TValue? tPrevValue)? equalityFn,
```

Comment:
One thing I'm still planning on addressing; I think this typing should probably be 
```suggestion
  bool Function(TValue tNextValue, TValue tPrevValue)? equalityFn,
```
but I need to confirm.

Reply:
Done in 8a37dcb32195eb69152371950633aa7d101d0885